libraries/app/include/graphene/app/api.hpp

```diff
@@ -112,16 +112,15 @@ namespace graphene { namespace app {
                                                               operation_history_id_type start = operation_history_id_type())const;
 
          /**
-          * @brief Get operations relevant to the specificed account
+          * @brief Get operations relevant to the specificed account, return a list of operation history objects with transaction id
```

Comment:
"Get operations relevant to the specificed account, return a list of operation history objects with transaction id"
should be something more like:
"Get operations relevant to the specified account filtering by operation type"
same everywhere.
the parameter name i think it should be "operation_types" instead of "operation_indexs" or "operation_ids".

rest looks good.

This code does not affect performance of the node so i don't have any reason for not to add except for the limit thing that is now specified as a max number ... still don't like much calling for 100 results and getting 1. much more better to do it with the elasticsearch plugin and wrapper as: http://209.188.21.157:5000/get_account_history?account_id=1.2.282&from=100&size=10&sort_by=-operation_type&operation_type=0

anyways, i am not going to oppose to it as there is not infrastructure for elasticsearch yet, if you guys think it is needed then lets make that small changes and add it. 

thanks @gxchain123

Reply:
another limitation of elasticsearch is that the cli_wallet cant make calls into it yet. i think that in the future the cli can access the database for specific calls like this but not present atm. thats another argument to merge this code at least by now.

Reply:
updated.